### ISSUE TITLE
Consider supporting subscription system

### ISSUE THREAD

**Issue by bluepeter** [pilwon/ultimate-seed#69]
Why not add a subscription system: with monthly and annual plans, free trials, coupons, etc. And some payment processor handling (Stripe & PayPal maybe?)

**Comment by pilwon** [comment]
@bluepeter That would be pretty sweet and useful.

@yaru22 What do you think?

**Comment by yaru22** [comment]
@pilwon @bluepeter Yeah, integrating Stripe would be a fantastic idea. I read Stripe doc recently and they added a few more neat features that I'd like to use for my projects, too. Also, Stripe took over node-stripe library (previously maintained by a third-party) and started maintaining it. So I think it's a good time to integrate that to ultimate-seed!

@pilwon, we can discuss further in person how we do this when I get back?

**Comment by pilwon** [comment]
:+1:

**Comment by DanieleSalatti** [comment]
Good idea, but make it easy to remove it: I will need to replace it with Paymill (the European counterpart).

**Comment by pilwon** [comment]
@DanieleSalatti Having second thoughts, I believe we should keep it outside the main seed. If we ever integrate Stripe with `ultimate-seed`, we will make it an add-on.
